### PR TITLE
C418594 Verify that final custom field can be removed (volaris) (TaaS)

### PR DESCRIPTION
**Pull request by KarapetyanSofia**
[C418594 ](https://foliotest.testrail.io/index.php?/cases/view/418594)

[Allure Report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5434/allure)
image

### CODE REVIEW

File: cypress/e2e/users/custom-fields/verify-final-custom-field-removed.cy.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      cy.wait(1000);
+      CustomFields.addCustomSingleSelect(singleSelectData);
+      cy.wait(1000);
+      CustomFields.deleteCustomField(fieldData.fieldLabel);
+      CustomFields.deleteCustomField(singleSelectData.data.fieldLabel);
+      CustomFields.deleteCustomField(checkboxData.fieldLabel);
+      cy.wait(1000);
+      CustomFields.editButton();
+      CustomFields.deleteLastCustomField();
+      CustomFields.isTrashButtonAbsent();
```
Comment (manowarqqq) [inline cypress/e2e/users/custom-fields/verify-final-custom-field-removed.cy.js:62]: Nee to ccheck that All custom fields are removed. Too abstract method's name

File: cypress/e2e/users/custom-fields/verify-final-custom-field-removed.cy.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    Users.deleteViaApi(user.userId);
+  });
+
+  it(
+    'C418594 Verify that final custom field can be removed (volaris) (TaaS)',
+    { tags: [TestTypes.extendedPath, DevTeams.volaris] },
+    () => {
+      cy.visit(TopMenu.customFieldsPath);
+      CustomFields.addCustomTextField(fieldData);
+      cy.wait(1000);
```
Comment (manowarqqq) [inline cypress/e2e/users/custom-fields/verify-final-custom-field-removed.cy.js:51]: Please remove unnesessary waits

File: cypress/e2e/users/custom-fields/verify-final-custom-field-removed.cy.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      CustomFields.addCustomCheckBox(checkboxData);
+      cy.wait(1000);
+      CustomFields.addCustomSingleSelect(singleSelectData);
+      cy.wait(1000);
```
Comment (manowarqqq) [inline cypress/e2e/users/custom-fields/verify-final-custom-field-removed.cy.js:55]: In Step 5 need to verify that At least 3 custom fields are present on the 3rd pane view

**Review by manowarqqq — CHANGES_REQUESTED** [review]

File: cypress/e2e/users/custom-fields/verify-final-custom-field-removed.cy.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+      secondLabel: `autotestSecondLabel_${getRandomPostfix()}`,
+    },
+  };
+  const checkboxData = {
+    fieldLabel: `autotestFieldLabel_${getRandomPostfix()}`,
+    helpText: `autotestHelpText_${getRandomPostfix()}`,
+  };
+  before('Create test data', () => {
+    cy.getAdminToken();
+    cy.createTempUser([Permissions.uiUsersCustomField.gui, Permissions.uiUsersCreate.gui]).then(
```
Comment (manowarqqq) [inline cypress/e2e/users/custom-fields/verify-final-custom-field-removed.cy.js:29]: Redundant
cy.createTempUser alredy has cy.getAdminToken

File: cypress/support/fragments/settings/users/customFields.js [hunk 5]
```diff
@@ -8,6 +8,7 @@ import {
   Pane,
   TextField,
 } from '../../../../../interactors';
+import label from '../../../../../interactors/label';
```
Comment (manowarqqq) [inline cypress/support/fragments/settings/users/customFields.js:11]: Please remove unused import

**Review by manowarqqq — COMMENTED** [review]
Please, run tests that used method you chanded, to be ensure it doesn't affect
C15693
C389568
C389570

File: cypress/e2e/users/custom-fields/verify-final-custom-field-removed.cy.js (hunk 4 above)
Comment (KarapetyanSofia) [reply cypress/e2e/users/custom-fields/verify-final-custom-field-removed.cy.js:29]: Done!

**Review by manowarqqq — APPROVED** [review]

File: cypress/support/fragments/settings/users/customFields.js [hunk 6]
```diff
@@ -44,11 +44,22 @@ export default {
     // Wait for changes to be saved and reflected
     cy.wait(15000);
   },
-
+  createViaApi(field) {
```
Comment (re-jas) [inline cypress/support/fragments/settings/users/customFields.js:47]: Why do you need this method? I can't find you using this.

Comment (KarapetyanSofia) [reply cypress/support/fragments/settings/users/customFields.js:47]: I delete this method, also the cy.getAdminToken(); but as I see it`s the same, I`ll do one more time!

**Review by re-jas — APPROVED** [review]

**Pull request merged**